Package.swift

```diff
@@ -20,7 +20,15 @@ func resolveTargets() -> [Target] {
 let package = Package(
     name: "RevenueCat",
     platforms: [
-        .macOS(.v10_12), .iOS(.v9), .watchOS("6.2"), .tvOS(.v9)
+        .macOS(.v10_12),
+        .watchOS("6.2"),
+        .tvOS(.v9),
+
+        // todo: deployment_target set to 12.0 instead of 9.0 temporarily for iOS due to a known issue in 
+        // Xcode-beta 5, where swift libraries fail to build for iOS targets that use armv7.
+        // See issue 74120874 in the release notes:
+        // https://developer.apple.com/documentation/xcode-release-notes/xcode-13-beta-release-notes
```

Comment:
I had tested with the previous release of beta 2, I just re-tested with the new `13C5081f` build and I get the same result. 

For context, what happens is that you get this message whenever the min OS version is set to < 12.0: 

```ruby
    - ERROR | [iOS] xcodebuild:  /Users/andresboedo/repos/purchases-ios/Purchases/Support/ManageSubscriptionsModalHelper.swift:136:23: error: cannot find 'AppStore' in scope
    - ERROR | [iOS] xcodebuild:  /Users/andresboedo/repos/purchases-ios/Purchases/Purchasing/ProductDetails.swift:23:40: error: no type named 'Product' in module 'StoreKit'
    - ERROR | [iOS] xcodebuild:  /Users/andresboedo/repos/purchases-ios/Purchases/Purchasing/StoreKit2/ProductsFetcherSK2.swift:39:46: error: module 'StoreKit' has no member named 'Product'
    - ERROR | [iOS] xcodebuild:  /Users/andresboedo/repos/purchases-ios/Purchases/Purchasing/StoreKit2/StoreKit2TransactionListener.swift:39:42: error: no type named 'Product' in module 'StoreKit'
    - ERROR | [iOS] xcodebuild:  /Users/andresboedo/repos/purchases-ios/Purchases/Purchasing/StoreKit2/SK2BeginRefundRequestHelper.swift:62:21: error: no type named 'Transaction' in module 'StoreKit'
    - ERROR | [iOS] xcodebuild:  /Users/andresboedo/repos/purchases-ios/Purchases/Purchasing/StoreKit2/SK2BeginRefundRequestHelper.swift:100:55: error: no type named 'Transaction' in module 'StoreKit'
    - ERROR | [iOS] xcodebuild:  /Users/andresboedo/repos/purchases-ios/Purchases/Purchasing/StoreKit2/SK2BeginRefundRequestHelper.swift:122:62: error: no type named 'Transaction' in module 'StoreKit'
    - ERROR | [iOS] xcodebuild:  /Users/andresboedo/repos/purchases-ios/Purchases/Purchasing/StoreKit2/SK2BeginRefundRequestHelper.swift:37:33: error: module 'StoreKit' has no member named 'Transaction'
    - ERROR | [iOS] xcodebuild:  /Users/andresboedo/repos/purchases-ios/Purchases/Purchasing/StoreKit2/SK2BeginRefundRequestHelper.swift:64:39: error: module 'StoreKit' has no member named 'Transaction'
    - ERROR | [iOS] xcodebuild:  /Users/andresboedo/repos/purchases-ios/Purchases/Purchasing/StoreKit2/SK2BeginRefundRequestHelper.swift:81:53: error: no type named 'Transaction' in module 'StoreKit'
    - ERROR | [iOS] xcodebuild:  /Users/andresboedo/repos/purchases-ios/Purchases/Purchasing/StoreKit2/StoreKit2TransactionListener.swift:64:36: error: cannot find type 'VerificationResult' in scope
    - ERROR | [iOS] xcodebuild:  /Users/andresboedo/repos/purchases-ios/Purchases/Purchasing/StoreKit2/StoreKit2TransactionListener.swift:81:39: error: no type named 'Transaction' in module 'StoreKit'
    - ERROR | [iOS] xcodebuild:  /Users/andresboedo/repos/purchases-ios/Purchases/Purchasing/StoreKit2/StoreKit2TransactionListener.swift:33:33: error: module 'StoreKit' has no member named 'Transaction'
    - ERROR | [iOS] xcodebuild:  /Users/andresboedo/repos/purchases-ios/Purchases/Purchasing/TrialOrIntroPriceEligibilityChecker.swift:79:35: error: type of expression is ambiguous without more context
```

Upon further testing, this might be a separate bug, because setting min deployment target to iOS 11 works, and the suggested workaround for the linked radar says to use 12. Setting to 10 or 9 will output the error above. 

The easiest way to test this out is to update the deployment target in the Podspec and run `pod lib lint`. 

@NachoSoto If you have some time, would you mind taking a look? I'd appreciate a sanity check.

Reply:
We talked about this in private and as per the discussion, we're dropping support for older iOS versions.